--- a/.vite/build/workspace-root-drop-handler-BeuIN9UA.js
+++ b/.vite/build/workspace-root-drop-handler-BeuIN9UA.js
@@ -40118,6 +40118,7 @@
     i = t.bi(e.version);
   return (
     vz({
+      enabled: !1,
       dsn: t.yi,
       environment: Dz.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -71557,6 +71557,7 @@
   let n = iH(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (Sk({
+    enabled: !1,
     dsn: hG,
     environment: e.buildFlavor,
     release: gG(n.version),
