--- a/webview/assets/app-initial~app-main~page-BF1QkwFT.js
+++ b/webview/assets/app-initial~app-main~page-BF1QkwFT.js
@@ -72928,11 +72928,15 @@
       (pQe = 5),
       (mQe = 500),
       (s5 = {
+        overrideAdapter: window.__ELECTRON_SHIM__.overrideAdapter,
+        logLevel: 0,
+        disableLogging: !0,
         networkConfig: {
           api: uQe,
           logEventUrl: I8,
           sdkExceptionUrl: dQe,
           networkOverrideFunc: JZe,
+          preventAllNetworkTraffic: !0,
         },
       }),
       (hQe = 1));
